test/e2e/network/proxy.go

```diff
@@ -77,19 +77,30 @@ var _ = SIGDescribe("Proxy", func() {
 			    Description: Ensure that proxy on node logs works with generic top
 				level prefix proxy and explicit kubelet port.
 		*/
-		if !skipPrefixProxyTests {
-			framework.ConformanceIt("should proxy logs on node with explicit kubelet port ", func() { nodeProxyTest(f, prefix+"/proxy/nodes/", ":10250/logs/") })
-		}
+		framework.ConformanceIt("should proxy logs on node with explicit kubelet port ", func() {
+			if skipPrefixProxyTests {
```

Comment:
the value is set in BeforeEach, so is only valid to check inside the It() blocks